pkgs/development/python-modules/pivy/default.nix

```diff
@@ -11,6 +11,8 @@ buildPythonPackage rec {
     sha256 = "0vids7sxk8w5vr73xdnf8xdci71a7syl6cd35aiisppbqyyfmykx";
   };
 
+  dontUseCmakeConfigure = true;
+
   nativeBuildInputs = with pkgs; [
     swig qmake cmake
```

Comment:
You are sure that both cmake and qmake are required?

Reply:
In the next release, the build system will change to cmake so I'm not very inclined to change anything now.

But yes, `qmake` is required.  The build script calls `qmake -query` to check whether qt is installed.